contracts/StablecoinConverter.sol

```diff
@@ -209,20 +215,43 @@ contract StablecoinConverter is EpochTokenLocker {
                 executedSellAmount
             );
         }
-        checkAndOverrideObjectiveValue(uint(tokenConservation[0]));
+        int objectiveValue = utility - disregardedUtility;
```

Comment:
Intuitiviely, I would go for uint instead of int for the objective value.

Because:
1. I can always submit the trivial solution without orders, which will have an objective value of 0
2. Any valuable solution in the 2-dimensional case has a positive utility- disregarded utility.
3. overflow issue handling with ints is much harder.

But for sure, we should check it more in detail

Reply:
Right, yes the trivial solution is considered "better" than other non-trivial ones with a lot of disregarded utility. This, kind-of, implies nobody would submit such a solution... because someone else would immediately submit the trivial solution which is better. In this case I would impose the following requirement

```js
require(utility >= disregardedUtility, "solution must be better than trivial");
```

Reply:
This is unfortunately not going to work as expected, since some of our unit tests have fairly large negative objective values.